src/MediaWiki/Hooks.php

```diff
@@ -342,6 +341,11 @@ private function registerHandlers() {
 			'AdminLinks' => [ $this, 'onAdminLinks' ],
 			'PageSchemasRegisterHandlers' => [ $this, 'onPageSchemasRegisterHandlers' ]
 		];
+		if ( version_compare( MW_VERSION, '1.35', '<' ) ) {
```

Comment:
Remember that we do not need to support MW older than 1.35 on master

Reply:
thanks, fixing.